### DISCUSSION TITLE
Source of depth maps

### DISCUSSION THREAD

**Discussion by rockywind** [OPEN-AIR-SUN/mars#18] [General]
Hi，thank you for sharing your great work!
Can you sharing the depth completion script?
````
    dataparser_outputs = self._generate_dataparser_outputs(split)
  File "/rockywin.wang/NeRF/mars/nsg/data/nsg_dataparser.py", line 1040, in _generate_dataparser_outputs
    image_filenames, depth_name, semantic_name = get_scene_images_tracking(
  File "/rockywin.wang/NeRF/mars/nsg/data/nsg_dataparser.py", line 622, in get_scene_images_tracking
    for frame_no in range(len(os.listdir(left_depth_path))):
FileNotFoundError: [Errno 2] No such file or directory: '/rockywin.wang/NeRF/mars/data/kitti_tracking/training/completion_02/0006'
````

**Comment by wuzirui** [comment]
Hi, the depth maps we use are actually generated with a [monocular depth estimation model](https://github.com/theNded/mini-omnidata). Sorry for the misleading names.

**Reply by xuyanging** [reply]
hi，you used monocular depth estimation model to get depth map other than kitti official depth label, right ?

**Comment by rockywind** [comment]
Why do not use the sparse depth from lidar projection?

**Comment by wuzirui** [comment]
We tried depth completion for the KITTI dataset at first (and thus the name completion_02/03 hah). Don't know whether there's anything wrong with our evaluation scripts, the generated completions are not even as good as the monocular estimated ones. If there are any suggestions, PRs or comments are welcomed.

**Comment by programath** [comment]
> Hi, the depth maps we use are actually generated with a [monocular depth estimation model](https://github.com/theNded/mini-omnidata). Sorry for the misleading names.

Did you train the network from scratch (or finetune) on KITTI? Otherwise how did you deal with the scale ambiguity problem of monocular depth estimation?

**Comment by wuzirui** [comment]
Hi! Please take a look at the supplementary of our paper. We apply the scale-invariant loss if monocular depth is used.

Other references:
1. [MiDAS](https://github.com/isl-org/MiDaS)
2. [MonoSDF](https://niujinshuchong.github.io/monosdf/)

**Comment by pierremerriaux-leddartech** [comment]
Hi @wuzirui , where could I found the supplementary of our paper ? are you referring to your paper annex, section A, B...  ?
thanks

**Reply by JiantengChen** [reply]
Hi! Our supplementary material can be found following the references.
